drivers/vfs/copy_linux.go

```diff
@@ -0,0 +1,7 @@
+package vfs
+
+import "github.com/containers/storage/drivers/copy"
+
+func dirCopy(srcDir, dstDir string) error {
+	return copy.DirCopy(srcDir, dstDir, copy.Content, false)
```

Comment:
I'm not 100% that this should be `false` rather than `true`, but it's not a blocker.